src/test/integration/controllers/auth.spec.js

```
@@ -114,4 +114,142 @@
+      );
+      
+      const response = await app
+        .post('/auth/google')
+        .send({ idToken: 'valid_token' });
+
+      expect(response.status).toBe(500);
+    });
+  })
+
+  describe('Gmail Auth endpoint', () => {
+    it('should return 401 for invalid Gmail token', async () => {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)